cmd/aws-janitor/main.go

```diff
@@ -130,7 +132,13 @@ func main() {
 		logrus.Errorf("Error parsing --include-tags: %v", err)
 		runtime.Goexit()
 	}
-
+	skipResourceRecordSetTypesSet := map[string]bool{}
+	if len(skipResourceRecordSetTypes) == 0 {
+		skipResourceRecordSetTypes.Set("SOA,NS")
```

Comment:
A comment that these cannot be deleted would probably be great, to justify the default.